example/server/src/index.ts

```diff
@@ -19,7 +19,7 @@ const data = parse(readFileSync(resolve(cwd, "data/starwars_data.yaml"), "utf-8"
 const inigoCfg = new InigoConfig({
   Token: INIGO_SERVICE_TOKEN,
   Schema: typeDefs,
-  DisableResponseMerge: false,
```

Comment:
Should this field be called "DisableResponseData", according to the struct in the sidecar, or is there something I'm missing?

Reply:
renamed